lib/cli.js

```diff
@@ -60,13 +47,17 @@ function init() {
   env.lookup(function () {
     // list generators
     if (opts.generators) {
-      return console.log(utils.generatorsFromEnv(env).join('\n'));
+      return console.log(require('./utils').generatorsFromEnv(env).join('\n'));
     }
 
     // If no generator is passed, then start the Yo UI
     if (!cmd) {
       if (opts.help) {
-        return console.log(fs.readFileSync(path.join(__dirname, 'usage.txt'), 'utf8'));
+        return console.log(
+          require('fs').readFileSync(
+            require('path').join(__dirname, 'usage.txt'),  'utf8'
+          )
+        );
```

Comment:
then https://github.com/yeoman/yo/blob/master/lib/cli.js#L66-L68 should be fixed as well

What is the deciding factor on when to keep on one line vs multiple? I broke it up like this as it was a long line.